src/view/com/post-thread/PostThread.tsx

```diff
@@ -271,7 +271,7 @@ function PostThreadLoaded({
       } else if (item === REPLY_PROMPT && hasSession) {
         return (
           <View>
-            {isDesktop && <ComposePrompt onPressCompose={onPressReply} />}
+            {!isMobile && <ComposePrompt onPressCompose={onPressReply} />}
```

Comment:
Opted for `!isMobile` because the case where we show it at a fixed position uses `isMobile &&` as the condition. Ideally we'd unify those (so that it's always rendered from here) but whatever.